models/v1/job_status.go

```diff
@@ -29,20 +31,20 @@ type JobStatus struct {
 	// Created timestamp
 	//
 	// required: true
-	// example: 2006-01-02T15:04:05Z
-	Created string `json:"created"`
+	// swagger:strfmt date-time
+	Created time.Time `json:"created"`
```

Comment:
Created is the time the K8S job was created. It can be nil when radix-operator does an intermediate status update when looping through jobs to create in the spec.